src/main/java/teammates/ui/webapi/request/InstructorPrivilegeUpdateRequest.java

```diff
@@ -0,0 +1,75 @@
+package teammates.ui.webapi.request;
+
+import javax.annotation.Nullable;
+
+/**
+ * The update request for instructor privilege.
+ */
+public class InstructorPrivilegeUpdateRequest extends BasicRequest {
+
+    @Nullable
+    private String sectionName;
+
+    @Nullable
+    private String feedbackSessionName;
+
+    private boolean canModifyCourse;
+    private boolean canModifySession;
+    private boolean canModifyStudent;
+    private boolean canModifyInstructor;
+
+    private boolean canViewStudentInSections;
+
+    private boolean canModifySessionCommentsInSections;
+    private boolean canViewSessionInSections;
+    private boolean canSubmitSessionInSections;
+
+    @Override
+    public void validate() {
```

Comment:
I think you can validate the hierarchy here.

1. If `section` == null && `session` == null, only general privileges should present.
2. If xxx then xxx
...